regtests/bin/matrix.base

```diff
@@ -85,6 +85,7 @@
   echo "  echo '      Multi 07 (unstr. + reg. grds)      : $multi07'"          >> matrix.head
   echo "  echo '      Multi 08 (with ice)                : $multi08'"          >> matrix.head
   echo "  echo '      Assim Update of the restart file   : $assim'"            >> matrix.head
+  echo "  echo '      Atmosphere, ocean, and ice coupling: $coup'"             >> matrix.head
```

Comment:
I'd prefer that we specify that these are OASIS coupling as there are multiple ways to couple and this is only for OASIS testing.